### PR TITLE
Initial game setup

### PR DESCRIPTION
**Pull request by gurtatiLND**
- made the initial setup of the game(HTML, CSS and JS files, and assets folder);
- added some logic and initial page view;

Please review my PR and leave feedback.
Thank you.

### CODE REVIEW

File: README.md
Comment (lopezelpesado) [inline README.md:1]: Please don't delete the README 😭

**Review by lopezelpesado — CHANGES_REQUESTED** [review]

File: README.md
Comment (lopezelpesado) [reply README.md:1]: I think I know what's happened here. The easiest thing would probably be to save these files somewhere, start fresh with a new branch off main (make sure your main branch is up to date), add your files and make a new pull request 👍

Comment (gurtatiLND) [reply README.md:1]: OK, almost done

**Comment by lopezelpesado** [comment]
Looks like you managed to get your code in with #138! Nice one and thank you for not deleting the `README.md`!

**Pull request closed without merging**